src/Drivers/VectoryLinkDriver.php

```diff
@@ -52,29 +53,27 @@ public function message(string $message): string
 
     public function send(): SMSDriverResponseInterface
     {
-        $response = HTTP::get($this->endPoint, $this->headers(), $this->payload());
+        $response = HTTP::post($this->endPoint, $this->headers(), $this->payload());
```

Comment:
@Mostafa7000 Hello Mostafa. Could you provide us with the documentation you are using to change the HTTP method

Reply:
The documentation I found online for VictoryLink's APIs is fairly outdated: from 2020.
Changing request format from JSON to x-www-form-urlencoded is done after I searched for other repositories that work with VL, mainly this one: _https://github.com/bellalMohamed/victory-link-sms-api_.

Trying the new format worked like a charm for us. Before, we had regular outages in the service.
This is the API we are currently using: https://smsvas.vlserv.com/KannelSending/service.asmx/SendSMS.

I just sent to VL support team to get an updated documentation.